test/fixtures/static_assets_app/app/templates/home/dashboard.html.erb

```diff
@@ -0,0 +1,2 @@
+dashboard
+
```

Comment:
Trailing new line should be removed :) 

Also maybe change the text to something like "This is a dashboard page, used to test static assets" ?